extensions/amp-accordion/0.1/amp-accordion.js

```diff
@@ -601,6 +609,30 @@ class AmpAccordion extends AMP.BaseElement {
       tryFocus(newFocusHeader);
     }
   }
+
+  /**
+   * Callback function to execute when mutations are observed.
+   * @param {!Array<!MutationRecord>} mutations
+   */
+  toggleMutations_(mutations) {
+    if (mutations.length > 1) {
+      return;
+    }
+    const mutation = mutations[0];
+    if (mutation.attributeName !== 'expanded') {
```

Comment:
Would be better to instead use the attribute filter from `MutationObserver`: https://developer.mozilla.org/en-US/docs/Web/API/MutationObserverInit/attributes

```javascript
this.mutationObserver_.observe(section, {
  attributes: true,
  attributeFilter: ['expanded'],
});
```

and name the observer something like `expandedObserver`.